meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -463,7 +475,17 @@ Panel {
                         Layout.fillHeight: true
                         horizontalAlignment: Text.AlignHCenter
                         verticalAlignment: Text.AlignVCenter
-                        text: "Add Images"
+                        text: {
+                            if (isMeshroomScene) {
+                                if(nbFilesDropped == 1) {
+                                    return "Load Project"
+                                } else {
+                                    return "Only one project"
+                                }
```

Comment:
This `if/else` can never be fully entered: if `isMeshroomScene` is true, then the number of files is necessarily 1 already.

So the "Only one project" can never be reached. The current behaviour is that nothing happens when hovering with more than 1 mg file. It might be better to count the number of .mg files / not .mg files contained in the dragging: if at least one file is not an .mg one, the counter of .mg file is reset, and that counter is what the message when hovering is based on.